### DISCUSSION TITLE
Project is not compiling to ES5

### DISCUSSION THREAD

**Discussion by arezooalipanah** [vercel/next.js#22168] [Help]
Some files inside my project are not compiled to ES5 when I build the project, for instance they contain const declarations. As far as I know Next automatically transpiles project to es5 so why this is happening? 
Here is like to the project so you can reproduce, and I'm using only one dependency, "react-hook-inview", but have tested with some other libs like "react-lazyload" and got the same issue.

My project link:
https://github.com/arezooalipanah/lazyload/tree/main

So my problem is that in browsers that do not support es6, the code will not run, for example, Chrome 48 gives this error
"Uncaught ReferenceError: Invalid left-hand side in assignment"

![WhatsApp Image 2021-02-14 at 17 31 07](https://user-images.githubusercontent.com/20445227/107879861-a1c23d80-6ef0-11eb-86d0-bab80b16bb30.jpeg)

![Capture](https://user-images.githubusercontent.com/20445227/107879900-d46c3600-6ef0-11eb-8b23-18c8df30107a.PNG)

**Comment by l0ger** [comment]
Hi Arezoo,

NextJS doesn't transpire to Es5  itself. It uses Babel. Babel doesn't transpile your code to Es5. Babel transpile your code to browser-compatible code, not Es5. The modern browser understands const keyword. open your browser and define the const variable to see that.
